grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy

```diff
@@ -176,7 +176,7 @@ class StockMovementApiController {
             try {
                 stockTransferService.deleteStockTransfer(params.id)
             } catch (Exception e) {
-                def message = "Unable to delete stock movement with ID ${params.id}: " + e.message
+                def message = "${warehouse.message(code: 'stockMovement.delete.error.message')}"
```

Comment:
We want to get rid of `warehouse.message` at some points, so let's use `g.message` if possible. Consider adding a default message also.
```suggestion
                def message = "${g.message(code: 'stockMovement.delete.error.message')}"
```